doc/admin.md

```diff
@@ -415,7 +416,13 @@ IotAgents, as all Express applications that use the body-parser middleware, have
 size that the application will handle. This default limit for ioiotagnets are 1Mb. So, if your IotAgent receives a
 request with a body that exceeds this limit, the application will throw a “Error: Request entity too large”.
 
-The 1Mb default can be changed setting the `expressLimit` configuration parameter (or equivalente `IOTA_EXPRESS_LIMIT` environment variable).
+The 1Mb default can be changed setting the `expressLimit` configuration parameter (or equivalente `IOTA_EXPRESS_LIMIT`
+environment variable).
+
+#### `storeLastMeasure`
+
+If this flag is activated, last measure arrived to Device IoTAgent without be processed will be stored in Device under
+lastMeasure field with in timestamp. This flag is disabled by default.
```

Comment:
Maybe an example on how the measure is stored would be helpful. By the "will be stored in Device under
lastMeasure field with in timestamp" it is not fully clear (e.g. is a sub-JSON object used for that)?

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1670/commits/1f6f3f4f481082c84f8970669e4c816aa5df5c24